Apply-For-Me-Api/src/main/java/com/hydraulic/applyforme/repository/jpa/ProfessionalJpaRepository.java

```diff
@@ -10,4 +10,5 @@ public interface ProfessionalJpaRepository extends JpaRepository<Professional, L
 
     @Query("select p from Professional p where p.member.id = :id")
     Professional getProfessional(@Param("id") Long memberId);
+    Professional getProfessionalByMemberEmailAddress(String emailAddress);
 }
```

Comment:
reflect the above comment on this also. we do not need to show the personal details of user on the request url